Objects/moduleobject.c

```diff
@@ -866,6 +866,40 @@ _Py_module_getattro(PyModuleObject *m, PyObject *name)
     return _Py_module_getattro_impl(m, name, 0);
 }
 
+static int
+module_setattro(PyModuleObject *m, PyObject *name, PyObject *value)
+{
+    PyObject *setattr, *delattr;
```

Comment:
Please move the variable declaration where they are assigned for the first time.